deploy/operator.yaml

```diff
@@ -12,10 +12,9 @@ spec:
       labels:
         name: descheduler-operator
     spec:
-      serviceAccountName: descheduler-operator
       containers:
         - name: descheduler-operator
-          image: registry.svc.ci.openshift.org/openshift/origin-v4.0:descheduler-operator
+          image: docker.io/ravig/descheduler-operator:v0.0.7
```

Comment:
you mean to commit this?

Reply:
Nope, this would have been removed, I was using it for testing. But I want to see, if we can build  images and push it to external registry for PR.

Reply:
yeah, i'm not sure how to integrate with CI (and OLM inside CI for that matter) to inject the image-under-test here.

Reply:
I just spoke to @ecordell from OLM team on slack. Seems they are working on it.